### ISSUE TITLE
Make failed due t a so called "fac0109aa2acf14469320b049d710dd42639bf94" --repo 

### ISSUE THREAD

**Issue by JeongHan-Bae** [pyodide/pyodide-build#35]
I just pulled a new pyodide to my wsl
i made it
showing
"""
Requirement already satisfied: pyodide-cli~=0.2.1 in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (0.2.4)
Collecting pyodide-lock==0.1.0a6 (from pyodide-build==0.27.3)
  Obtaining dependency information for pyodide-lock==0.1.0a6 from https://files.pythonhosted.org/packages/d6/3b/562cfda37d814351049f4ee3ac154de6e8e46f8f2c7fa7b5fe1698409f72/pyodide_lock-0.1.0a6-py3-none-any.whl.metadata
  Using cached pyodide_lock-0.1.0a6-py3-none-any.whl.metadata (2.2 kB)
Requirement already satisfied: requests in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (2.32.3)
Requirement already satisfied: resolvelib in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (1.0.1)
Requirement already satisfied: rich in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (13.8.1)
Requirement already satisfied: ruamel-yaml in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (0.18.6)
Requirement already satisfied: typer in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (0.12.5)
Requirement already satisfied: unearth~=0.6 in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (0.17.2)
Requirement already satisfied: virtualenv in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (20.26.5)
Requirement already satisfied: wheel in /home/master/.local/lib/python3.12/site-packages (from pyodide-build==0.27.3) (0.44.0)
Requirement already satisfied: leb128 in /home/master/.local/lib/python3.12/site-packages (from auditwheel-emscripten~=0.0.9->pyodide-build==0.27.3) (1.0.8)
Requirement already satisfied: pyproject_hooks in /home/master/.local/lib/python3.12/site-packages (from build~=1.2.0->pyodide-build==0.27.3) (1.1.0)
Requirement already satisfied: annotated-types>=0.6.0 in /home/master/.local/lib/python3.12/site-packages (from pydantic<3,>=2->pyodide-build==0.27.3) (0.7.0)
Requirement already satisfied: pydantic-core==2.23.4 in /home/master/.local/lib/python3.12/site-packages (from pydantic<3,>=2->pyodide-build==0.27.3) (2.23.4)
Requirement already satisfied: typing-extensions>=4.6.1 in /home/master/.local/lib/python3.12/site-packages (from pydantic<3,>=2->pyodide-build==0.27.3) (4.12.2)
Requirement already satisfied: httpx<1,>=0.27.0 in /home/master/.local/lib/python3.12/site-packages (from unearth~=0.6->pyodide-build==0.27.3) (0.27.2)
Requirement already satisfied: charset-normalizer<4,>=2 in /home/master/.local/lib/python3.12/site-packages (from requests->pyodide-build==0.27.3) (3.3.2)
Requirement already satisfied: idna<4,>=2.5 in /home/master/.local/lib/python3.12/site-packages (from requests->pyodide-build==0.27.3) (3.10)
Requirement already satisfied: urllib3<3,>=1.21.1 in /home/master/.local/lib/python3.12/site-packages (from requests->pyodide-build==0.27.3) (2.2.3)
Requirement already satisfied: certifi>=2017.4.17 in /home/master/.local/lib/python3.12/site-packages (from requests->pyodide-build==0.27.3) (2024.8.30)
Requirement already satisfied: markdown-it-py>=2.2.0 in /home/master/.local/lib/python3.12/site-packages (from rich->pyodide-build==0.27.3) (3.0.0)
Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /home/master/.local/lib/python3.12/site-packages (from rich->pyodide-build==0.27.3) (2.18.0)
Requirement already satisfied: ruamel.yaml.clib>=0.2.7 in /home/master/.local/lib/python3.12/site-packages (from ruamel-yaml->pyodide-build==0.27.3) (0.2.8)
Requirement already satisfied: click>=8.0.0 in /home/master/.local/lib/python3.12/site-packages (from typer->pyodide-build==0.27.3) (8.1.7)
Requirement already satisfied: shellingham>=1.3.0 in /home/master/.local/lib/python3.12/site-packages (from typer->pyodide-build==0.27.3) (1.5.4)
Requirement already satisfied: distlib<1,>=0.3.7 in /home/master/.local/lib/python3.12/site-packages (from virtualenv->pyodide-build==0.27.3) (0.3.8)
Requirement already satisfied: filelock<4,>=3.12.2 in /home/master/.local/lib/python3.12/site-packages (from virtualenv->pyodide-build==0.27.3) (3.16.1)
Requirement already satisfied: platformdirs<5,>=3.9.1 in /home/master/.local/lib/python3.12/site-packages (from virtualenv->pyodide-build==0.27.3) (4.3.6)
Requirement already satisfied: anyio in /home/master/.local/lib/python3.12/site-packages (from httpx<1,>=0.27.0->unearth~=0.6->pyodide-build==0.27.3) (4.6.0)
Requirement already satisfied: httpcore==1.* in /home/master/.local/lib/python3.12/site-packages (from httpx<1,>=0.27.0->unearth~=0.6->pyodide-build==0.27.3) (1.0.5)
Requirement already satisfied: sniffio in /home/master/.local/lib/python3.12/site-packages (from httpx<1,>=0.27.0->unearth~=0.6->pyodide-build==0.27.3) (1.3.1)
Requirement already satisfied: h11<0.15,>=0.13 in /home/master/.local/lib/python3.12/site-packages (from httpcore==1.*->httpx<1,>=0.27.0->unearth~=0.6->pyodide-build==0.27.3) (0.14.0)
Requirement already satisfied: mdurl~=0.1 in /home/master/.local/lib/python3.12/site-packages (from markdown-it-py>=2.2.0->rich->pyodide-build==0.27.3) (0.1.2)
Using cached pyodide_lock-0.1.0a6-py3-none-any.whl (9.9 kB)
Building wheels for collected packages: pyodide-build
  Building wheel for pyodide-build (pyproject.toml) ... done
  Created wheel for pyodide-build: filename=pyodide_build-0.27.3-py3-none-any.whl size=238395 sha256=da5295ea9d2fd5a7cf3b167596966e329792a41fbdc5e16ae8ff5321cd282556
  Stored in directory: /home/master/.cache/pip/wheels/ee/15/b9/d0948667e0881e82572e6e0bfdda8b254fc918811316ae6f58
Successfully built pyodide-build
Installing collected packages: pyodide-lock, pyodide-build
  Attempting uninstall: pyodide-lock
    Found existing installation: pyodide-lock 0.1.0a7
    Uninstalling pyodide-lock-0.1.0a7:
      Successfully uninstalled pyodide-lock-0.1.0a7
  Attempting uninstall: pyodide-build
    Found existing installation: pyodide-build 0.29.0
    Uninstalling pyodide-build-0.29.0:
      Successfully uninstalled pyodide-build-0.29.0
Successfully installed pyodide-build-0.27.3 pyodide-lock-0.1.0a6

[notice] A new release of pip is available: 23.2.1 -> 24.2
[notice] To update, run: pip3.12 install --upgrade pip
make[1]: *** [Makefile:232: pyodide_build] Error 1
make[1]: Leaving directory '/home/master/projects/pyodide'
make: *** [Makefile:239: dist/python_stdlib.zip] Error 2
"""
i did "pip cache purge"
i made sure to have the newest pip
always showing
"""make[1]: Entering directory '/home/master/projects/pyodide'
Ensuring required pyodide-build version is installed
./tools/check_and_install_pyodide_build.py "fac0109aa2acf14469320b049d710dd42639bf94" --repo "https://github.com/pyodide/pyodide-build"
make[1]: *** [Makefile:232: pyodide_build] Error 1
make[1]: Leaving directory '/home/master/projects/pyodide'
make: *** [Makefile:239: dist/python_stdlib.zip] Error 2
"""

no way could i solve the problem
i could not fetch the "fac0109aa2acf14469320b049d710dd42639bf94" repo

i just successfully built the program earlier this day but now all is ruined

**Comment by ryanking13** [comment]
The error appears to be caused by the `pyodide` executable not being found.

What's the content inside ` /home/master/.local/bin`? Installing pyodide-build should put `pyodide` executable (entry point) in that directory (based on your log).

**Comment by agriyakhetarpal** [comment]
Yes, and if your virtual environment wrapper does not place executables in the bin directory for some reason, you could also try running `python -m pyodide build <...>`.

**Comment by ryanking13** [comment]
Closing as no reply, but don't hesitate to reopen if you still have the same issue.
